src/js/view/cinema.js

```diff
@@ -0,0 +1,66 @@
+import starFilled from '../../assets/star_filled.png';
+const IMAGE_BASE_URL = 'https://image.tmdb.org/t/p/w500';
+
+export class Cinema {
+    #movies = [];
+    page = 1;
+    #moviesWrapper = document.querySelector('.item-list');
+    #skeletonWrapper = document.querySelector('.skeleton-list');
+
+    setMovies(movies) {
+        this.#movies = movies;
+    }
+
+    showMovies() {
+        const newItemsHTML = this.#movies
+            .map((movieInfo) => {
+                return `
+              <li>
+                <a href="#">
+                  <div class="item-card">
+                    <img
+                      class="item-thumbnail"
+                      src="${IMAGE_BASE_URL}${movieInfo.getPosterPath()}"
+                      loading="lazy"
+                      alt="${movieInfo.title}"
+                    />
+                    <p class="item-title">${movieInfo.getTitle()}</p>
+                    <p class="item-score">
+                    ${movieInfo.getVoteAverage()} <img src="${starFilled}" alt="별점" /> 
+                    </p>
+                  </div>
+                </a>
+              </li>
+            `;
+            })
+            .join('');
+        this.#moviesWrapper.innerHTML += newItemsHTML;
```

Comment:
```suggestion
        const newItemsHTML = this.#movies
            .map((movieInfo) => `
              <li>
                <a href="#">
                  <div class="item-card">
                    <img
                      class="item-thumbnail"
                      src="${IMAGE_BASE_URL}${movieInfo.getPosterPath()}"
                      loading="lazy"
                      alt="${movieInfo.title}"
                    />
                    <p class="item-title">${movieInfo.getTitle()}</p>
                    <p class="item-score">
                    ${movieInfo.getVoteAverage()} <img src="${starFilled}" alt="별점" /> 
                    </p>
                  </div>
                </a>
              </li>
            `
            ).join('');
        this.#moviesWrapper.innerHTML += newItemsHTML;
```
return문은 제거해도 무방해보이네요!